app/views/layouts/application.html.haml

```diff
@@ -5,5 +5,8 @@
     = javascript_pack_tag 'application'
     %title .portrait.
   %body
-    %h1 portrait
-    #Content= yield
+    %h1= link_to 'portrait', root_path
```

Comment:
Was annoying to try and navigate the site without a way to get back to the home page (would have to use back buttons repeatedly)

So made the "logo" a link to homepage